### PR TITLE
Tag CovarianceMatrices v0.1.2 and v0.2.0

### PR DESCRIPTION
**Pull request by gragusa**
- v0.1.2 is the last release to support Julia v0.4
- v0.2.0 minor release Julia v0.5 only

### CODE REVIEW

File: CovarianceMatrices/versions/0.1.2/requires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+julia 0.4 0.5-
+StatsBase 
+GLM 0.5.5
+DataFrames
+Compat
```
Comment (tkelman) [inline CovarianceMatrices/versions/0.1.2/requires]: should declare a minimum version that implements the feature you need

**Comment by gragusa** [comment]
Hopefully this will fix things.

File: CovarianceMatrices/versions/0.1.2/requires (hunk 1 above)
Comment (tkelman) [inline CovarianceMatrices/versions/0.1.2/requires]: still need a minimum version on the 0.1.2 tag

File: CovarianceMatrices/versions/0.2.0/requires [hunk 2]
```diff
@@ -2,4 +2,3 @@ julia 0.5
 StatsBase 
 GLM 0.6
 DataFrames
-Compat
```
Comment (tkelman) [inline CovarianceMatrices/versions/0.2.0/requires]: you're still doing `import Compat.view` on master which you'll need to remove for this to work

Comment (gragusa) [inline CovarianceMatrices/versions/0.2.0/requires]: Removed `import Compat.view` from master, retagged and updated sha of the tag. 

For the v0.1.2 tag, the tag REQUIRES 0.8.4 of compat, but I forget to update the REQUIRE of the METADATA PR. Now I did it.

**Comment by gragusa** [comment]
Thanks!

**Pull request merged**